packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -58,7 +65,11 @@ class FederationRuntimePlugin {
   constructor(options?: moduleFederationPlugin.ModuleFederationPluginOptions) {
     this.options = options ? { ...options } : undefined;
     this.entryFilePath = '';
-    this.bundlerRuntimePath = options?.embedRuntime
+    this.bundlerRuntimePath = this.getBundlerRuntimePath();
+  }
+
+  getBundlerRuntimePath() {
+    return this.options?.embedRuntime
       ? VendoredBundlerRuntimePath
```

Comment:
Can we infer the VendoredBundlerRuntimePath from options.implementation if user set the value

Reply:
I think we can if we can resolve the package name or base path - but i think i can avoid using vendored paths with the newer container hoisting of modules - since i can depend on more modules and polyfills with better tracking. I can try removing this

Reply:
How would this look? I already do this lower down. 
```
    if (this.options?.implementation) {
      const runtimePath = this.options.embedRuntime
        ? '@module-federation/webpack-bundler-runtime/vendor'
        : '@module-federation/webpack-bundler-runtime';
      this.bundlerRuntimePath = require.resolve(runtimePath, {
        paths: [this.options.implementation],
      });
    }
```

Reply:
that's nice ! lgtm